stable/insights-agent/templates/polaris/cronjob.yaml

```diff
@@ -26,15 +27,20 @@ spec:
               subPath: config.yaml
               readOnly: true
             {{ end }}
-            command: [
-              "polaris", "audit",
-              {{ if .Values.polaris.config }}
-              "--config", "/opt/app/config.yaml",
-              {{ end }}
-              "--output-file", "/output/polaris.json",
-              "--format", "json",
-              "--display-name", "{{ .Values.insights.cluster }}",
-            ]
+            command:
+            - "sh"
+            - "-c"
+            - |
+              CONFIG=""
+              if [ -f '/output/polaris-config.yaml' ]
+              then
+                CONFIG='--config /output/polaris-config.yaml'
+              fi
+              if [ -f '/opt/app/config.yaml' ]
+              then
+                CONFIG='--config /opt/app/config.yaml'
+              fi
```

Comment:
So we always run the download script, and if it 404s we use the default config. Is that right?